itou/metabase/management/commands/_job_applications.py

```diff
@@ -14,7 +14,7 @@
 
 def get_job_application_origin(ja):
     if ja.sender_kind == JobApplication.SENDER_KIND_PRESCRIBER:
-        if ja.sender_prescriber_organization and ja.sender_prescriber_organization.is_authorized:
+        if ja.sender_prescriber_organization and ja.is_sent_by_authorized_prescriber:
```

Comment:
Il me semble que tu peux simplifier non?

`if ja.is_sent_by_authorized_prescriber:`

Reply:
Of course! C'est mon `search and replace` qui a travaillé mais j'avais pas vu ce cas. Merci